tox.ini

```diff
@@ -115,3 +115,35 @@ env_dir = {work_dir}/pre-commit
 runner = ignore_env_name_mismatch
 description = Run pre-commit on staged files or arbitrary pre-commit commands (tox run -e pre-commit -- [args])
 commands = pre-commit {posargs:run}
+
+[docs]  # Sphinx documentation configuration
+extras = docs
+package = editable
+no_package = true
+env_dir = {work_dir}/docs
+runner = ignore_env_name_mismatch
+source_dir = {tox_root}/{project_name}
+
+[testenv:build-docs]
```

Comment:
Looking at the CI for this PR, this reminds me of something I did in Snapcraft that I haven't upstreamed: https://github.com/canonical/snapcraft/blob/676e391446225901b5a09cfbfb4d11c186ce752d/.github/workflows/tox.yaml#L42

The problem is that if a command's docstring contains invalid RST text, it won't get caught by `lint-docs` in CI.  The biggest offender in Snapcraft was single backticks instead of double backticks.

Think it's worthwhile to upstream to starbase?

Reply:
Probably worth it with a comment about why we're building them :-)